models/migrations/v207.go

```diff
@@ -67,7 +67,7 @@ func addWebAuthnCred(x *xorm.Engine) error {
 				Name:            reg.Name,
 				LowerName:       strings.ToLower(reg.Name),
 				UserID:          reg.UserID,
-				CredentialID:    base64.RawStdEncoding.EncodeToString(parsed.KeyHandle),
+				CredentialID:    base32.HexEncoding.EncodeToString(parsed.KeyHandle),
```

Comment:
either do v208 or this!

(I like the v208 approach more as it will not hurt our `next` users)

Reply:
(we however could do a noop 208 and patchend v207 as followup)

that way migrations from "latest stable" tags will encode once and `next` users have no issue either

Reply:
I don't think we can no-op either unfortunately.

Reply:
```suggestion
				CredentialID:    base64.RawStdEncoding.EncodeToString(parsed.KeyHandle),
```

Reply:
Sorry I've just seen what you were meaning! I don't know how I ended up changing the v207.go migration here - that very much unintended!